README-vi.md

```diff
@@ -127,34 +127,54 @@ Review the [Contributing Guidelines](CONTRIBUTING.md).
 </p>
 
 * [System design topics: start here](#system-design-topics-start-here)
+* [Các chủ đề thiết kế hệ thống: bắt đầu ở đây](#system-design-topics-start-here)
     * [Step 1: Review the scalability video lecture](#step-1-review-the-scalability-video-lecture)
+    * [Bước 1: Xem qua các video bài giảng về scalability](#step-1-review-the-scalability-video-lecture)
     * [Step 2: Review the scalability article](#step-2-review-the-scalability-article)
+    * [Bước 2: Xem qua các bài viết về scalability](#step-2-review-the-scalability-article)
     * [Next steps](#next-steps)
-* [Các chủ đề thiết kế hệ thống: bắt đầu từ đây](#system-design-topics-start-here)
+    * [Bước tiếp theo](#next-steps)
 * [Performance vs scalability](#performance-vs-scalability)
+* [Hiệu suất và scalability]
 * [Latency vs throughput](#latency-vs-throughput)
+* [Độ trễ và công suất](#latency-vs-throughput)
 * [Availability vs consistency](#availability-vs-consistency)
+* [Availability và tính nhất quán](#availability-vs-consistency)
     * [CAP theorem](#cap-theorem)
+    * [Định lý CAP](#cap-theorem)
         * [CP - consistency and partition tolerance](#cp---consistency-and-partition-tolerance)
+        * [CP - tính nhất quán và partition tolerance](#cp---consistency-and-partition-tolerance)
         * [AP - availability and partition tolerance](#ap---availability-and-partition-tolerance)
+        * [AP - availability và partition tolerance](#ap---availability-and-partition-tolerance)
```

Comment:
I think it's better to be:
[AP - Tính sẵn có và dung sai phần vùng](#ap---availability-and-partition-tolerance)